TLM/CSUtil.Commons/Log.cs

```diff
@@ -62,8 +61,8 @@ private enum LogLevel {
                     File.Delete(LogFilePath); // delete old file to avoid confusion.
                 }
 
-                var args = Environment.GetCommandLineArgs().ToList();
-                int index = args.IndexOf("-logFile");
+                var args = Environment.GetCommandLineArgs();
+                int index = Array.FindIndex(args, x => x == "-logFile");
```

Comment:
this is still allocating a delegate
you can do `(args as IList<string>).IndexOf("-logFile");`

but either way I don't care!

Reply:
I wish C# did not require me to cast array as IList to use IList extension methods.
I don't have to cast it to IEnumerable to use IEnumerable extension methods for that.

Reply:
you cannot add/remove stuff from array so maybe that is why.
but still array can be casted to IList.
and trying to add remove stuff throws exception:
```
(x as IList<string>).Add("zarrin")
System.NotSupportedException: Collection was of a fixed size.
```